TradeItIosTicketSDK2/TradeItSelectBrokerViewController.swift

```diff
@@ -206,7 +206,7 @@ class TradeItSelectBrokerViewController: CloseableViewController, UITableViewDel
             if let broker = broker {
                 let brokerLogoImageView = UIImageView()
                 if TradeItBrokerLogoService.setLogo(forBroker: broker, onImageView: brokerLogoImageView, withSize: .small) {
-                    if let cell = tableView.dequeueReusableCell(withIdentifier: "TRADE_IT_FEATURED_BROKER_CELL_ID") as? TradeItYahooFeaturedBrokerTableViewCell {
+                    if let cell = tableView.dequeueReusableCell(withIdentifier: "TRADE_IT_FEATURED_BROKER_CELL_ID") as? TradeItFeaturedBrokerTableViewCell {
```

Comment:
Has the storyboard been updated with the custom class TradeItFeaturedBrokerTableViewCell?